src/ui/Banner/Banner.jsx

```diff
@@ -1,22 +1,68 @@
 import React from 'react';
 import { Container, Icon, Button, Grid } from 'semantic-ui-react';
 import PropTypes from 'prop-types';
+import { formatDate } from '@plone/volto/helpers/Utils/Date';
+import config from '@plone/volto/registry';
 
 Banner.propTypes = {
   title: PropTypes.string,
   image: PropTypes.bool,
 };
 
-function Banner({ image_url, image, children }) {
+const socialPlatforms = {
+  facebook: {
+    shareLink: (url) => `https://facebook.com/sharer.php?u=${url}`,
+  },
+  twitter: {
+    shareLink: (url) => `https://www.twitter.com/share?url=${url}`,
+  },
+  linkedin: {
+    shareLink: (url) =>
+      `https://www.linkedin.com/sharing/share-offsite/?url=${url}`,
+  },
+  reddit: {
+    shareLink: (url, title) => `https://reddit.com/submit?url=${url}`,
+  },
+};
+
+export const getImageSource = (image) => {
+  if (image?.download) return image.download;
```

Comment:
@nileshgulia1 with this pull request the homepage gif is no longer rendering.
This is what the URL is rendered with this branch which is incorrect
background-image:url(https://demo-www.eea.europa.eu/en/jumping-bug.gif)
This is what is currently which works
background-image: url("https://demo-www.eea.europa.eu/en/jumping-bug.gif/@@images/image");

Reply:
@ichim-david You need to test this PR with https://github.com/eea/volto-eea-website-theme/pull/76. Sorry for not mentioning beforehand.